pages/Hypr Ecosystem/hypridle.md

```diff
@@ -30,31 +32,50 @@ Example listener:
 
 ```ini
 listener {
-    timeout = 500                            # in seconds
-    on-timeout = notify-send "You are idle!" # command to run when timeout has passed
+    timeout = 500                            # in seconds.
+    on-timeout = notify-send "You are idle!" # command to run when timeout has passed.
     on-resume = notify-send "Welcome back!"  # command to run when activity is detected after timeout has fired.
 }
 ```
 
 You can define as many listeners as you want.
 
-Full hypridle example with hyprlock
+Full hypridle example with hyprlock:
 
 ```ini
+general {
+    lock_cmd = pidof hyprlock || hyprlock       # avoid starting multiple hyprlock instances.
+    before_sleep_cmd = loginctl lock-session    # lock before suspend.
+    after_sleep_cmd = hyprctl dispatch dpms on  # to avoid having to press a key twice to turn on the display.
+}
+
+# set monitor backlight to minimum
+listener {
+    timeout = 150                                # 2.5min.
+    on-timeout = brightnessctl -s set 0          # monitor backlight off.
```

Comment:
I wouldn't set this to 0 as on OLEDs that will actually mean literally 0

Reply:
so, OLEDs users can escape monitor backlight listener?

Reply:
no, I'd just set it to like 10% instead